k8s/demo/common/bsp/bulk-scan-processor.yaml

```diff
@@ -0,0 +1,33 @@
+---
+apiVersion: helm.fluxcd.io/v1
+kind: HelmRelease
+metadata:
+  name: bulk-scan-processor
+  namespace: bsp
+  annotations:
+    flux.weave.works/automated: "true"
+    flux.weave.works/tag.java: glob:prod-*
+spec:
+  releaseName: bulk-scan-processor
+  rollback:
+    enable: true
+    retry: true
+  chart:
+    repository: https://hmctspublic.azurecr.io/helm/v1/repo/
+    name: bulk-scan-processor
+    version: 0.2.19
+  values:
+    java:
+      replicas: 1
+      useInterpodAntiAffinity: true
+      cpuRequests: '500m'
+      cpuLimits: '2500m'
+      image: hmctspublic.azurecr.io/bulk-scan/processor:prod-f01fe889
```

Comment:
you can use :latest

Reply:
Does it make any difference if this is going to be updated automatically?

Reply:
my guess would be "just in case it will not be tagged something as _prod-..._"

Reply:
What you you mean, Don?

Reply:
i mean in case that tag is not found and flux never deploys anything and maybe never auto-updates. just hypothesis here. i have no problem with what it is now ;)

Reply:
I thought it may first deploy the one stated in the config with even there is a newer version. it should be fine that's why I approved :)

Reply:
Is this correct?
```
image: hmctspublic.azurecr.io/bulk-scan/processor:latest
```

Reply:
yes, after deployment you will see flux updated with the image tag

Reply:
Thanks for explaining, guys. Done.